### PR TITLE
PMM-12522 collect chunks metrics in low resolution

### PR DESCRIPTION
**Pull request by BupycHuk**
[PMM-12522](https://jira.percona.com/browse/PMM-12522) (optional, if ticket reported)

- [ ] Links to related pull requests (optional).
https://github.com/percona/pmm/pull/2987

---

Below we provide a basic checklist of things that would make it a good PR:
- Make sure to sign the CLA (Contributor License Agreement).
- Make sure all tests pass.
- Keep current with the target branch and fix conflicts if necessary.
- Update jira ticket description if necessary.
- Attach screenshots and/or console output to the jira ticket to confirm new behavior, if applicable.
- Leave notes to the reviewers if you need to focus their attention on something specific.

Once all checks pass and the code is ready for review, please add `pmm-review-exporters` team as the reviewer. That would assign people from the review team automatically. Report any issues on our [Forum](https://forums.percona.com).

[PMM-12522]: https://perconadev.atlassian.net/browse/PMM-12522?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by JiriCtvrtka — APPROVED** [review]

File: exporter/shards_collector.go [hunk 1]
```diff
@@ -56,6 +57,28 @@ func (d *shardsCollector) collect(ch chan<- prometheus.Metric) {
 	client := d.base.client
 	logger := d.base.logger
 	prefix := "shards collection chunks"
+	ctx := d.ctx
+
+	metrics := make([]prometheus.Metric, 0)
+	logger.Debugf("chunksTotal")
```
Comment (ademidoff) [inline exporter/shards_collector.go:63]:
```suggestion
	logger.Debugf("creating chunksTotal")
```

File: exporter/shards_collector.go [hunk 2]
```diff
@@ -56,6 +57,28 @@ func (d *shardsCollector) collect(ch chan<- prometheus.Metric) {
...
+	metric, err := chunksTotal(ctx, client)
+	if err != nil {
+		logger.Warnf("cannot create metric for chunks total: %s", err)
+	} else {
+		metrics = append(metrics, metric)
+	}
+
+	logger.Debugf("chunksTotalPerShard")
```
Comment (ademidoff) [inline exporter/shards_collector.go:71]:
```suggestion
	logger.Debugf("creating chunksTotalPerShard")
```

File: exporter/shards_collector.go (hunk 1 above)
Comment (ademidoff) [reply exporter/shards_collector.go:63]: Just to make the message a bit more useful )

File: exporter/exporter.go [hunk 3]
```diff
@@ -267,7 +267,7 @@ func (e *Exporter) getClient(ctx context.Context) (*mongo.Client, error) {
 	return client, nil
 }
 
-// Handler returns an http.Handler that serves metrics. Can be used instead of
+// Handler returns a http.Handler that serves metrics. Can be used instead of
```
[the remark this answers is not in the document]
Comment (BupycHuk) [reply exporter/exporter.go:270]: that was a suggestion from IDE :) googled and reverting

[the remark this answers is not in the document]
Comment (ademidoff) [reply exporter/exporter.go]: Stupid IDE 😃

**Review by ademidoff — APPROVED** [review]

**Pull request merged**